androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/surveydetail/Spinner.kt

```diff
@@ -14,15 +14,17 @@ import com.chargemap.compose.numberpicker.ListItemPicker
 fun Spinner(
     modifier: Modifier = Modifier,
     surveyAnswerUiModels: List<SurveyAnswerUiModel>,
+    onAnswerSelected: (surveyAnswerUiModel: SurveyAnswerUiModel) -> Unit,
 ) {
     val answerTexts = surveyAnswerUiModels.map { it.text }
     var answerText by remember { mutableStateOf(answerTexts[0]) }
```

Comment:
Used the object itself instead of `text` or `id` as the value. https://github.com/nimblehq/avishek-phong-kmm-ic/pull/138/commits/e3b80bf489c7572d7b2545e52be8fc7e6b0e2d42